### PR TITLE
Added multiple tab support

### PR DESCRIPTION
**Pull request by Itisha2987**

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -1,121 +1,124 @@
...
       var update_acc = null;
 
-      // Gmail
-      const gmail_regex = "https:\/\/mail.google.com\/mail\/u\/[0-9].*";
-      if (current_url.match(gmail_regex)) {
-        update_url_regex = RegExp("u\/[0-9]");
-        update_acc = "u/" + account_num
-      }
+        // Gmail
+        const gmail_regex = "https:\/\/mail.google.com\/mail\/u\/[0-9].*";
```
Comment (anshulahuja98) [inline background.js:10]: There has been an unnecessary change in indentation for the whole block, please revert his, the git diffs generated would spoil the commit history.

File: background.js [hunk 2]
```diff
@@ -1,121 +1,124 @@
...
+         if (current_url.match(keep_regex)) {
+           update_url_regex = RegExp("u\/[0-9]");
+           update_acc = "u/" + account_num
+         }
+  
+        if (update_acc && update_url_regex) {
+          current_url = current_url.replace(update_url_regex, update_acc);
+          console.log(current_url)
+          if(command.substring(0,command.length - 1)=="alt")
+          chrome.tabs.update({ url: current_url });
```
Comment (anshulahuja98) [inline background.js:113]: Here where it should've been indented has not been indented

File: key_event.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+if (window == top) {
+    window.addEventListener('keyup', doKeyPress, false); //add the keyboard handler
+    }
+    
+    var acc_num = [0,1,2,3,4,5,6,7,8,9]
```
Comment (anshulahuja98) [inline key_event.js:5]: Please clean up the code

File: key_event.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    function doKeyPress(e){
+	var n = parseInt(e.keyCode)-48
+        if (((e.shiftKey&&e.altKey)||e.altKey)&&acc_num.includes(n)){
```
Comment (anshulahuja98) [inline key_event.js:8]: Here as well

File: manifest.json [hunk 5]
```diff
@@ -7,61 +7,19 @@
     "scripts": ["background.js"],
     "persistent": false
   },
-  "permissions": [
-    "tabs"
-  ],
+
+  "permissions": ["http://*/*", "tabs"],
```
Comment (anshulahuja98) [inline manifest.json:11]: Do we also not need permissions for https separately?

File: key_event.js (hunk 4 above)
Comment (Itisha2987) [reply key_event.js]: I have made the required changes, waiting for further comments.

File: key_event.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+if (window == top){
+    window.addEventListener('keyup', doKeyPress, false);
+}
+    
+var acc_num = [0,1,2,3,4,5,6,7,8,9]
+function doKeyPress(e){
+    var n = parseInt(e.keyCode)-48
+    if(((e.shiftKey&&e.altKey)||e.altKey)&&acc_num.includes(n)){
```
Comment (anshulahuja98) [inline key_event.js:8]: No Indentation?
Please use some automatic linter in your IDE

Comment (Itisha2987) [reply key_event.js]: Corrected, check it plz.
